modules/map/relative_map/navigation_lane.cc

```diff
@@ -425,8 +425,17 @@ bool NavigationLane::ConvertNavigationLineToPath(const int line_index,
       if (length > FLAGS_max_len_from_navigation_line) {
         return true;
       }
+      int path_size = navigation_path.path_point_size();
+      int path_size_ahead = path_size;
+#ifdef __aarch64__
+      path_size_ahead = current_project_index +
+                        FLAGS_relative_map_path_frame_ahead;
+      if (path_size_ahead > path_size) {
+        path_size_ahead = path_size;
+      }
+#endif
```

Comment:
Make the logic shorter and cleaner, such as:
```
      int path_size = navigation_path.path_point_size();
#ifdef __aarch64__
      int path_size_ahead = std::min(current_project_index + FLAGS_relative_map_path_frame_ahead, path_size);
#else 
      int path_size_ahead = path_size;
#endif
```

Reply:
#ifdef __aarch64__
      const int path_size_ahead = std::min(
          current_project_index + FLAGS_relative_map_path_frame_ahead,
          navigation_path.path_point_size());
#else 
      const int path_size_ahead = navigation_path.path_point_size();
#endif